src/http.js

```diff
@@ -72,9 +72,10 @@ export default class HTTP {
    * @param  {String} url               The URL.
    * @param  {Object} [options={}]      The fetch() options object.
    * @param  {Object} [options.headers] The request headers object (default: {})
+   * @param  {Object} [options.retry]   Number of retries (default: 1)
    * @return {Promise}
    */
-  request(url, options={headers:{}}) {
+  request(url, options={headers:{}, retry: 1}) {
```

Comment:
Shouldn't we default `retry` to `0`, so by default there's no retry attempt on failure like previously?